src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -535,7 +536,7 @@ def load_arguments(self, _):
             c.argument('tags', arg_type=tags_type)
             c.argument('external_gossip_certificates', nargs='*', validator=validate_gossip_certificates, options_list=['--external-gossip-certificates', '-e'], help="A list of certificates that the managed cassandra data center's should accept.")
             c.argument('cassandra_version', help="The version of Cassandra chosen.")
-            c.argument('authentication_method', arg_type=get_enum_type(['None', 'Cassandra']), help="Authentication mode can be None or Cassandra. If None, no authentication will be required to connect to the Cassandra API. If Cassandra, then passwords will be used.")
+            c.argument('authentication_method', arg_type=get_enum_type(['None', 'Cassandra', 'Ldap']), help="Authentication mode can be None or Cassandra. If None, no authentication will be required to connect to the Cassandra API. If Cassandra, then passwords will be used.")
```

Comment:
Documentation needs to note that it can be None, Cassandra, or Ldap

Reply:
updated documentation text.